pricing.html

```
@@ -122,7 +121,7 @@ <h3>Optional Add-On:</h3>
           <div class="plan-header">
             <div class="plan-icon">🤝</div>
             <h2>Partner</h2>
-            <div class="plan-price">$20,000<span>/month</span></div>
+            <div class="plan-price">Contact for pricing</span></div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </span> ]

(tag-pair)